fmcapi/api_objects/policy_services/natrules.py

```diff
@@ -25,26 +26,43 @@ def __init__(self, fmc, **kwargs):
         logging.debug("In __init__() for NatRules class.")
         self.parse_kwargs(**kwargs)
 
-    def nat_policy(self, name):
+    def nat_policy(self, nat_name=None, nat_id=None):
         """
         Associate NAT Policy.
 
         :param name: (str) Name of NAT Policy.
+        :param id: (str) ID of NAT Policy.
```

Comment:
Should be `nat_id` ;-)

Reply:
This should be corrected with the changed variable names for the nat_policy and lines up with acp in other classes.